tests/components/zwave_js/test_sensor.py

```diff
@@ -199,6 +199,15 @@ async def test_energy_sensors(
     assert state.attributes[ATTR_DEVICE_CLASS] == SensorDeviceClass.CURRENT
 
 
+async def test_basic_cc_sensor(
+    hass: HomeAssistant, client, multisensor_6, integration
```

Comment:
We should use a different example than the Multisensor 6, since the driver shouldn't expose the basic value for this device.